build/localnet/cluster-manager/main.go

```diff
@@ -33,7 +33,7 @@ var (
 )
 
 func init() {
-	rpcURL = fmt.Sprintf("http://%s:%s", runtime.GetEnv("RPC_HOST", "v1-validator001"), defaults.DefaultRPCPort)
+	rpcURL = fmt.Sprintf("http://%s:%s", runtime.GetEnv("RPC_HOST", "validator-001-pocket-validator"), defaults.DefaultRPCPort)
```

Comment:
Use the `Validator1EndpointK8S` constant you defined elsewhere.

Reply:
Great idea, especially with `defaults` already imported.